dandi/cli/cmd_ls.py

```diff
@@ -358,7 +360,15 @@ def fn():
                             digest=Digest.dandi_etag(digest),
                         ).json_dict()
                 else:
-                    rec = get_metadata(path)
+                    if use_fake_digest:
+                        digest = "0" * 32 + "-1"
+                    else:
+                        lgr.info("Calculating digest for %s", path)
+                        if path.endswith((".zarr", ".ZARR")):
```

Comment:
@TheChymera Also `.ngff`.  Use `ZARR_EXTENSIONS` from `dandi.consts`.

Reply:
Is there a reason why `ZARR_EXTENSIONS` is defined as a list and not a tuple in the first place?

Reply:
No, but you can just do `path.lower().endswith(tuple(ZARR_EXTENSIONS))`.

Reply:
yep, apready tupled it, will keep like that if you don't want to change the original var to tuple... `.lower()` is in fact not needed and I was wrong to include `".ZARR"` originally, since the only valid zarr files we have are BIDS, and BIDS is case-sensitive and only has lowercase zarr extensions.